integration_tests/conftest.py

```diff
@@ -17,13 +17,24 @@ def pytest_addoption(parser):
             PROFILE_DEVEL
         )
     )
+    parser.addoption(
+        "--no-cleanup",
+        action="store_true",
+        default=False,
+        help="Save the coverage report in `kcov_output` directory"
```

Comment:
I would be more explicit here:

`Keep the coverage report in kcov_output directory. If this flag is not provided,
both coverage related directories are removed.`

Reply:
Thanks! Updated.